### ISSUE TITLE
Getting "failed to update settings: invalid type: map, expected a boolean" in log for "deno.suggest.imports.hosts"

### ISSUE THREAD

**Issue by johanbrook** [sublimelsp/LSP-Deno#10]
Hi! Thanks for a great package 💕 

I'm 99% up and running with Deno in Sublime now, but I have this "One Last Thing"...

This is my project file:

```json
{
    "settings": {
        "lsp_format_on_save": true,
        "LSP": {
            "LSP-typescript": {
                "enabled": false,
            },
            "Deno": {
                "enabled": true,
                "settings": {
                    "deno.enable": true,
                    "deno.config": "./deno.jsonc",
                    "deno.unstable": true,
                    "deno.suggest.imports.hosts": {
                        "https://deno.land": true,
                        "https://cdn.skypack.dev": true,
                        "https://esm.sh": true,
                    }
                }
            },
        },
    },
}
```

When saving this, I get this message in the LSP log panel:

```
failed to update settings: invalid type: map, expected a boolean
```

This indicates some violation of the JSON schema, I assume.

No warning border is shown in the UI though. If I change some other property, the `LSP-json` package will complain and show a yellow border at the location of the incorrect property.

**If I remove the `deno.suggest.imports.hosts` property, the log line disappears.**

But to me, the schema in this package seems correct, when checking https://github.com/sublimelsp/LSP-Deno/blob/main/sublime-package.json#L140 ?

<details>
<summary>Full log</summary>

```
Deno: Starting Deno language server...
Deno:   version: 1.18.1 (release, x86_64-apple-darwin)
Deno:   executable: xxx/.deno/bin/deno
Deno: Connected to "Sublime Text LSP" 1.15.0
Deno:   Auto-resolved configuration file: "file:///xxx/deno.jsonc"
Deno: language server initialized
Deno: Server ready.
Deno: failed to update settings: invalid type: map, expected a boolean
Deno:   Auto-resolved configuration file: "file:///xxx/deno.jsonc"
```

</details>

Let me know if this is not the correct location for this issue. Thanks!

**Comment by rwols** [comment]
Probably the same bug as https://github.com/sublimelsp/LSP/issues/1858

**Comment by roblav96** [comment]
The **`LSP: Dump Window Configs`** command outputs:
```json
{
	"imports": {
		"hosts": {
			"https://deno": { "land": true },
			"https://cdn": { "skypack": { "dev": true } },
			"https://esm": { "sh": true }
		},
		"autoDiscover": false
	}
}
```

**Comment by rwols** [comment]
Should be fixed by LSP 1.16.3.
